src/xdocs/google_style.xml

```diff
@@ -339,7 +339,7 @@
                             <td><a name="3.3.3"/><a href="#3.3.3"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s3.3.3-import-ordering-and-spacing">3.3.3 Ordering and spacing</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s3.3.3-import-ordering-and-spacing">3.3.3 Ordering and spacing</a>
                             </td>
```

Comment:
@romani I don't think we support `a single blank line separates the two blocks. There are no other blank lines between import statements.` yet.
See top of https://github.com/checkstyle/checkstyle/pull/3519#issue-186128820 .